docs/source/_json/collection.json

```diff
@@ -5,62 +5,62 @@ HTTP 200 OK
 content-type: application/json
 
 {
-  "@context": "http://www.w3.org/ns/hydra/context.jsonld", 
```

Comment:
It's not obvious to me why this file has changes. The examples are generated when the tests are executed and have a trailing whitespace. Did you remove the trailing whitespaces manually?